docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -0,0 +1,165 @@
+
+Each rule type defines additional variables as properties of the variable `context`.  For example, if a rule type defines a variable `value`, it can be used in an action parameter as `{{context.value}}`.  
+
+For diagnostic or exploratory purposes, action variables whose values are objects, such as `context`, can be referenced directly as variables.  The resulting value will be a JSON representation of the object.  For example, if an action parameter includes `{{context}}`, it will expand to the JSON representation of all the variables and values provided by the rule type.
+
+You can attach more than one action. Clicking the "Add action" button will prompt you to select another rule type and repeat the above steps again.
+
+[role="screenshot"]
+image::images/rule-flyout-add-action.png[You can add multiple actions on a rule]
+
+[NOTE]
+==============================================
+Actions are not required on rules. You can run a rule without actions to understand its behavior, and then <<action-settings, configure actions>> later.
+==============================================
+
+[float]
+[[controlling-rules]]
+=== Mute and disable rules
+
+The rule listing allows you to quickly mute/unmute, disable/enable, and delete individual rules by clicking the action button. 
+
+[role="screenshot"]
+image:images/individual-mute-disable.png[The actions button allows an individual rule to be muted, disabled, or deleted]
+
+You can perform these operations in bulk by multi-selecting rules, and then clicking the *Manage rules* button:
+
+[role="screenshot"]
+image:images/bulk-mute-disable.png[The Manage rules button lets you mute/unmute, enable/disable, and delete in bulk,width=75%]
+
+[float]
+[[rule-details]]
+=== Drilldown to rule details
+
+Select a rule name from the rule listing to access the *Rule details* page, which tells you about the state of the rule and provides granular control over the actions it is taking. 
+
+[role="screenshot"]
+image::images/rule-details-alerts-active.png[Rule details page with three alerts]
+
+In this example, the rule detects when a site serves more than a threshold number of bytes in a 24 hour period. Three sites are above the threshold. These are called alerts - occurrences of the condition being detected - and the alert name, status, time of detection, and duration of the condition are shown in this view.
+
+Upon detection, each alert can trigger one or more actions. If the condition persists, the same actions will trigger either on the next scheduled rule check, or (if defined) after the re-notify period on the rule has passed. To prevent re-notification, you can suppress future actions by clicking on the eye icon to mute an individual alert. Muting means that the rule checks continue to run on a schedule, but that alert will not trigger any action.
```

Comment:
Is there an eye icon?  I don't see it in the images.

Also, suggest moving the sentenct "Muting means" to  here:

Mute and disable rules

The rule listing allows you to quickly mute/unmute, disable/enable, and delete individual rules by clicking the action button.  Muting means that the rule checks continue to run on a schedule, but that alert will not trigger any action.

Reply:
Updated in this commit: https://github.com/elastic/kibana/pull/101498/commits/97482fb695bc174811e512765b8a256c12ef96eb